keyboards/hlb/hyperbolas/hyp_s/keyboard.json

```diff
@@ -0,0 +1,492 @@
+{
+    "manufacturer": "HLB",
+    "keyboard_name": "hyperbolas",
+    "maintainer": "H3lli0n",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": [
+            "GP16", "GP17", "GP18", "GP19", "GP20", "GP21", "GP22", "GP23", "GP24",
+            "GP12", "GP6", "GP5", "GP4", "GP3", "GP25", "GP2", "GP1", "GP0"
+        ],
+        "rows": ["GP11", "GP10", "GP9", "GP8", "GP7", "GP15"]
+    },
+    "processor": "RP2040",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0007",
+        "vid": "0x3025"
```

Comment:
This vendor ID appears to be in use: 
https://www.the-sz.com/products/usbid/index.php?v=0x3025